src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.moyo__logo {
+  margin-top: 5px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, it seems the `.moyo__logo` class is intended for the logo, which is likely an `img` or `a` tag. Use a class to style it instead of the tag name.